webapp/src/Command/ImportEventFeedCommand.php

```diff
@@ -1064,161 +1017,64 @@ protected function importProblem(array $event): void
             'contest' => $this->contestId,
             'problem' => $problem,
         ]);
-        if ($contestProblem) {
-            $action = EventLogService::ACTION_UPDATE;
-        } else {
-            $this->logger->warning("No contest problem found in DOMjudge yet, creating one");
-            $contestProblem = new ContestProblem();
-            $contestProblem
-                ->setProblem($problem)
-                ->setContest($this->em->getRepository(Contest::class)->find($this->contestId));
-            $action = EventLogService::ACTION_CREATE;
-            $problem->addContestProblem($contestProblem);
-        }
-
-        if ($problem->getName() !== $event['data']['name']) {
-            $this->logger->warning(
-                'Problem name from DOMjudge (%s) does not match feed (%s)',
-                [ $problem->getName(), $event['data']['name'] ]
-            );
-            $problem->setName($event['data']['name']);
-        }
-
-        if (isset($event['data']['time_limit']) && abs($problem->getTimelimit() - $event['data']['time_limit']) > 0.001) {
-            $this->logger->warning(
-                'Time limit from DOMjudge (%.3f) does not match feed (%.3f)',
-                [ $problem->getTimelimit(), $event['data']['time_limit'] ]
-            );
-            $problem->setTimelimit($event['data']['time_limit']);
-        }
-
-        if ($contestProblem->getShortname() !== $event['data']['label']) {
-            $this->logger->warning(
-                'Contest problem shortname from DOMjudge (%s) does not match feed (%s)',
-                [ $contestProblem->getShortname(), $event['data']['label'] ]
-            );
-            $contestProblem->setShortname($event['data']['label']);
-        }
-
-        if ($contestProblem->getColor() !== ($event['data']['rgb'] ?? null)) {
-            $this->logger->warning(
-                'Contest problem color from DOMjudge (%s) does not match feed (%s)',
-                [ $contestProblem->getColor(), $event['data']['rgb'] ]
-            );
-            $contestProblem->setColor($event['data']['rgb'] ?? null);
+        if (!$contestProblem) {
+            $this->logger->warning("No contest problem found in DOMjudge yet");
```

Comment:
I read this as that there is no contest problem at all, whereas the code should say something like `Read contest problem does not exist in DOMjudge`?

Reply:
This is also resolved in a later commit.